src/utils/metadata/lists/blocklist-nsfw.json

```
@@ -88,6 +88,9 @@
   "nignogs",
   "nigs",
   "nonconsentual",
```

Should we do the same for `nonconsentual` for the typos?
```suggestion
  "nonconsentual",
  "non-consentual",
  "non consentual",
```